app/src/main/res/values-land/dimens.xml

```diff
@@ -450,8 +450,12 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">36dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">36dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">0dp</dimen>
-
+  
   <!-- OptionAudioLanguage -->
   <dimen name="option_audio_language_padding_start">36dp</dimen>
   <dimen name="option_audio_language_padding_end">36dp</dimen>
-</resources>
+
+  <!-- OptionsStoryTextSize -->
+  <dimen name="option_story_text_size_padding_start">36dp</dimen>
+  <dimen name="option_story_text_size_padding_end">36dp</dimen>
+  </resources>
```

Comment:
Please fix the indentation for last line. There should be no space at the start of resrouces.

Reply:
Done